main_test.go

```diff
@@ -80,7 +87,7 @@ func TestInitialTimeout(t *testing.T) {
 }
 
 func TestPrune(t *testing.T) {
-	cli, err := client.NewClientWithOpts()
+	cli, err := client.NewClientWithOpts(client.FromEnv)
```

Comment:
`client.FromEnv` was added so that Docker client configuration consumes docker configuration by environment variables.

Reply:
I think this should come in its own PR. As already hinted by you in Slack, probably a good idea to have a discussion of the configuration subsystem (/cc @mdelapenya).

Reply:
This configures the Docker client for Go package to reference the environment variables when loading the client. In deployed images, like the container it lives in, it shouldn't affect anything. However in a local/test environment, the client won't recognize the Docker context or environment configurations of `DOCKER_HOST` which eliminates the ability to run tests locally.

Reply:
Please ignore my comment, I was thinking in the context of `tc-go` rather than Ryuk.